templates/btax/includes/results/elasticity_table.html

```diff
@@ -0,0 +1,61 @@
+{% load humanize %}
```

Comment:
I agree we have more files to get rid of in `templates/btax/includes/results`.  I'll work on that.